dynatrace/api/v2/settings/objects/permissions/testdata/terraform/example_a.tf

```diff
@@ -0,0 +1,32 @@
+data "dynatrace_iam_user" "user_a" {
+  email = "a@home.com"
```

Comment:
Maybe we should change this one to `example.com`. We should not forward to any other sites (`home.com` redirects to `fairway.com`)